docs/lake/api.md

```diff
@@ -423,6 +423,7 @@ POST /query
 | query | string | body | Zed query to execute. All data is returned if not specified. ||
 | head.pool | string | body | Pool to query against Not required if pool is specified in query. |
 | head.branch | string | body | Branch to query against. Defaults to "main". |
+| ctrl | string | url | Set to "T" to include control messages in ZNG or ZJSON responses. Defaults to "F". |
```

Comment:
RFC 3986 calls this the query component of a URI.
```suggestion
| ctrl | string | query | Set to "T" to include control messages in ZNG or ZJSON responses. Defaults to "F". |
```

Reply:
Thanks @nwt. I must admit I'm slightly conflicted about that because "query" could potentially be mistaken as a reference a Zed query. But the example provides additional clarity so I'm happy to take the suggestion.